components/connectivity-adapter/internal/appregistry/service/handler.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/kyma-incubator/compass/components/connectivity-adapter/pkg/res"
 	"github.com/kyma-incubator/compass/components/director/pkg/graphql"
 	"github.com/kyma-incubator/compass/components/director/pkg/log"
+	"github.com/kyma-project/kyma/components/application-operator/pkg/normalization"
```

Comment:
I am not sure if code owner will be happy that code dependencies between kyma and compass is added.

Reply:
True. Do you suggest duplicating this package and pasting it also into this repository? What do you think about it, @kyma-incubator/cmp_team ?